my-dev-portal/src/components/pages/dashboard/OktaDashboard.js

```diff
@@ -0,0 +1,87 @@
+import { useOktaAuth } from "@okta/okta-react";
```

Comment:
hm.. I think it is possible to pass AuthInfo as a prop instead of have separate OktaDashboard vs Auth0Dashboard? 

Since dashboard shouldn't need to know which authenticatieon serivce. it just need to know if it is authenticated or not.

Reply:
but i know that it can take time to refactor and customer already wants this asap.